src/Nri/Ui/BorderlessButton/V1.elm

```diff
@@ -0,0 +1,114 @@
+module Nri.Ui.BorderlessButton.V1 exposing
+    ( button
+    , Size(..)
+    )
+
+{-|
+
+
+# About:
+
+BorderlessButton looks different from Nri.Ui.Button in that it displays without margin or padding.
+BorderlessButton has the suave, traditional look of a "link"!
+
+For accessibility purposes, buttons that perform an action on the current page should be HTML `<button>`
+elements and are created here with `*Button` functions. Buttons that take the user to a new page should be
+HTML `<a>` elements and are created here with `*Link` functions.
+
+
+# `<button>` creators
+
+@docs button
+
+
+# Config
+
+@docs Size
+
+-}
+
+import Css
+import Css.Global
+import Html.Styled as Html exposing (..)
+import Html.Styled.Events as Events
+import Nri.Ui
+import Nri.Ui.Colors.V1 as Colors
+import Nri.Ui.Fonts.V1
+import Nri.Ui.Icon.V4 as Icon exposing (IconType)
+
+
+{-| Sizes for the button
+-}
+type Size
+    = Small
+    | Medium
+    | Large
+
+
+{-| Config for the button
+-}
+type alias Config msg =
+    { label : String
+    , size : Size
+    , icon : Maybe IconType -- TODO: maybe do something else?
+    , onClick : msg
+    }
+
+
+{-| Creates a `<button>` element
+-}
+button : Config msg -> Html msg
+button config =
+    let
+        fontSize =
+            case config.size of
+                Small ->
+                    15
+
+                Medium ->
+                    17
+
+                Large ->
+                    20
+
```

Comment:
If this should behave like a link should it inherit the font size of the parent?